packages/dashboard/src/helpers/getUpdateConfig.ts

```diff
@@ -24,7 +24,13 @@ export const getUpdateConfig =
           _filter.values = filterValues
           if (filterValues.length > 0) {
             const defaultValues = _filter.pivot ? _filter.values : _filter.values[0]
-            _filter.active = _filter.active || defaultValues
+
+            const urlParams = new URLSearchParams(window.location.search);
+            if(_filter.setByQueryParameter && urlParams.get(_filter.setByQueryParameter) && _filter.values && _filter.values.includes(urlParams.get(_filter.setByQueryParameter))){

```

Comment:
Can you add a comment about what this is doing here or maybe abstract it out to a variable as below?
```const descriptionOfCondition = _filter.setByQueryParameter && urlParams.get(_filter.setByQueryParameter) && _filter.values && _filter.values.includes(urlParams.get(_filter.setByQueryParameter))

if(descriptionOFCondition) { // do something

Reply:
Moved to a shared function and addressed in latest commit